src/wallet/wallet_para_helpers.cpp

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (c) 2017 fireice-uk
+ * All rights reserved.
+ *
+ * Work-for-hire licence 
+ * Please read this licence carefully, as this is NOT a free software licence. 
+ * This agreement is designed to give you rights to preview and critique the work 
+ * for a limited amount of time before a payment is tendered. 
+ * 
+ * 1. Redistribution and use in source and binary forms WITHOUT modification is 
+ * permitted as long as this licence has not expired. You must not modify this work.
+ * 
+ * 2. As soon as the agreed upon payment is tendered, this licence expires and the 
+ * full exclusive copyright is transferred to The Monero Project, which is free to 
+ * re-release the work under whatever conditions they see fit.
+ * 
+ * 3. Unless point 2 comes into effect, this licence will expire on 01/08/2017. All 
```

Comment:
This could be mis-interpreted to believe that the license is already expired. US typically uses MM/DD/YYYY.

 Perhaps this could be changed to "August 1st, 2017".